Jenkinsfile

```diff
@@ -0,0 +1,37 @@
+node {
```

Comment:
Shouldn't this just live in `precious`?  It anyway _says_ "precious", so it's not particularly useful to someone outside of Binaris.

Reply:
That's how the flow goes for the other repos..